test/e2e/cypress/e2e/settings.cy.js

```diff
@@ -677,4 +677,133 @@ context('Settings page', () => {
       });
     });
   });
+
+  describe('Activity log', () => {
+    // Elements must be filtered to match only the ones relate to the Activity Logs.
```

Comment:
interesting approach.
Another way that I find helpful in order to get specific elements and avoid weird css selectors is using custom classes/labels/names (take a look at `SUMA Credentials Management` section)
No need to change anything, just wanting to share.

Reply:
First and foremost, I was experimenting thus I'm not sure _my_ approach is good.

We have to face 2 orders of problems here:
1. to select the element in the right section
2. to select an element that might have been re-rendered.

Regarding 1, maybe selecting even cypress docs says that selecting by text is not a best practice (https://docs.cypress.io/guides/references/best-practices#Text-Content). Perhaps we can assign a specific class/ID to every section? Quirk: models don't fall under any section :upside_down_face: 
I'm considering a refactor, though.

About 2, I found no other option than to use helper functions. I wonder if it's a huge burden for performance.

Reply:
Yes, I like that piece of document. I also find _using highly brittle selectors that are subject to change_ (which is not the case here) a pain :see_no_evil: 
All good from my side